<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="jobNameEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>450</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="priorityComboBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>510</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(173, 0, 0);</string>
    </property>
    <item>
     <property name="text">
      <string>High Demand</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Medium Demand</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Low Demand</string>
     </property>
    </item>
   </widget>
   <widget class="QSpinBox" name="delaySpinBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>480</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addJobButton">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>510</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Job</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="jobTable">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>431</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <column>
     <property name="text">
      <string>Timestamp</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Job Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Priority</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
